HW_7/task_34.py

```diff
@@ -0,0 +1,17 @@
+input_text = input("Write your song sounds: ")
+check_letters = list('аеёиоуыэюя')
+lett_list = []
+# input_text.split(" ")
+
+
+for each in input_text.split(" "):
```

Comment:
Если вам нужно поделить строку по пробелам, то лучше ничего методу `split` не передавать
```suggestion
for each in input_text.split():
```